desktop/cmp/panel/impl/dragger/DraggerModel.js

```diff
@@ -63,19 +64,19 @@ export class DraggerModel {
 
         const {clientX, clientY} = this.parseEventPositions(e);
         this.resizeState = {startX: clientX, startY: clientY};
-        this.startSize = panelModel.size;
+        this.startSize = panel[vertical ? 'offsetHeight' : 'offsetWidth'];
```

Comment:
We could have had this from the beginning I think.  Anyways, now it is necessary so as to avoid figure out true size from a % or px.